x-pack/test/functional/apps/maps/embeddable/dashboard.js

```diff
@@ -56,8 +56,10 @@ export default function ({ getPageObjects, getService }) {
     }
 
     it('should set "data-title" attribute', async () => {
-      const [{ title }] = await PageObjects.dashboard.getPanelSharedItemData();
```

Comment:
Instead of using a retry, how about adding some logic in before block to ensure dashboard has finished loading. So after line 37, await PageObjects.dashboard.loadSavedDashboard('map embeddable example');, add `await PageObjects.dashboard.waitForRenderComplete()`